site/app/templates/EditDisplayNameOrderForm.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-display-name-order-form{% endblock %}
+{% block title %}Edit Display Name Order{% endblock %}
+{% block body %}
+    <p class="std-margin">{{ "You can edit the order in which your name is displayed here!" }}</p>
+    <div class="flex-col" id="user_settings_modal">
+        <input id="display-name-order-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+        <h2>My Pronouns</h2>
+        <div class="edit_user_row ui-front">
+    <select name="display-name-order-change" id="display-name-order-change">
+        {% if user.getDisplayNameOrder() != 'REVERSED_F' %}
+        <option value="GIVEN_F">GIVEN_F</option>
```

Comment:
The option text should be more readable for the general audience like "Reversed" and "Given" instead of "REVERSED_F" and "GIVEN_F"